server/views/pages/supervisor/view-placement.njk

```diff
@@ -62,38 +68,41 @@
 <div class="govuk-box govuk-box--white govuk-box--rounded govuk-!-padding-top-1 govuk-!-padding-right-4 govuk-!-padding-left-4 govuk-!-margin-top-1">
 <div class="govuk-grid-row govuk-!-margin-top-4">
   <div class="govuk-grid-column-full">
-    {{ govukTable({
-      head: [
-        { text: "User ID" },
-        { text: "Name" },
-        { text: "Attended" },
-        { text: "Risk" },
-        { text: "Actions" }
-      ],
-      rows: [
-        [
-          { text: "P258053P" },
-          { text: "John Smith" },
-          { text: "7 hours" },
-          { html: mojBadge({ text: "HIGH ROSH 9.0", classes: "moj-badge--red" }) },
-          { html: '<a href="#" class="govuk-link">View</a> <a href="#" class="govuk-link">Remove</a>' }
-        ],
-        [
-          { text: "P258054P" },
-          { text: "Sarah Johnson" },
-          { text: "3 hours" },
-          { html: mojBadge({ text: "MEDIUM RSR 5.3", classes: "moj-badge--bright-purple" }) },
-          { html: '<a href="#" class="govuk-link">View</a> <a href="#" class="govuk-link">Remove</a>' }
+    {% if placement.attendees | length %}
+    {% set displayRows = [] %}
+{% for attendee in placement.attendees %}
+  {% set checkinLink = '<a href="/supervisor/check-in/' ~ placement.id ~ '/' ~ attendee.userId ~ '" class="govuk-link">Check-in</a>' %}
+  {% set checkoutLink = '<a href="/supervisor/check-out/' ~ placement.id ~ '/' ~ attendee.userId ~ '" class="govuk-link">Checkout</a>' %}
+
+  {% set actions = '<a href="#" class="govuk-link">View</a> <a href="/supervisor/remove/' ~ placement.id ~ '/' ~ attendee.userId ~ '" class="govuk-link">Remove</a>' %}
+
+  {% if attendee.status == 'Checked in' %}
+    {% set actions = actions + ' ' + checkoutLink %}
+  {% else %}
+    {% set actions = actions + ' ' + checkinLink %}
```

Comment:
Where is the "no show" link?
We also talked about adding a row to show how the "No Show" status would be presented (View only action)